### ISSUE TITLE
User Invitations not working!

### ISSUE THREAD

**Issue by jlaguma** [ONLYOFFICE/DocSpace#1665]
After compiling everything from source on ubuntu-22.04.3 server x86_64, user Invitations do not work. 
Front-end says it was sent in a Green Toast in top right, but actual invitation never arrives. 
Tried various email addresses. 

Where should I look for the issue?

Thanks.

**Comment by Aesuib** [comment]
The same with me, waiting for any answer,,,,,,,,

**Comment by Aesuib** [comment]
How did you compile from the source?
I installed it by using the official installing script, after installation, invitations do not work either, the same trouble, waiting for any answer too

**Comment by jlaguma** [comment]
After official installation using build/install/OneClickInstall/install-Docker.sh, everything works.
But if you manually compile everything using build/build.backend.docker.sh, everything works other than user invitations.
I think it's either a bug in build scripts, or I'm doing something wrong. I know it's because onlyoffice-notify docker container is not running. But not sure how to get it to run from compiled files.

**Comment by jlaguma** [comment]
Here is how I compile it from source step by step, maybe devs can point out my mistake:
1. install Ubuntu Server 22.04.3 x86_64
2. install yarn >=3
3. install dotnet sdk 7.0
4. install docker + docker-compose
5. install latest nvm and node 18
6. mkdir -p /app/onlyoffice
7. git clone --recurse-submodules -b master https://github.com/ONLYOFFICE/DocSpace.git /app/onlyoffice/src
8. modify /app/onlyoffice/src/build/build.backend.docker.sh
add "-sp /app/onlyoffice/src" to build-services.sh command
9. go into a build folder and replace "documentserver-de" to "documentserver" in .sh files so community version is compiled instead of development version
10. modify /app/onlyoffice/src/build/install/docker/.env and set
INSTALLATION_TYPE=COMMUNITY
11. modify /app/onlyoffice/src/build/build.backend.docker.sh and change the local_ip like to point to my ip:
local_ip=192.168.0.36
12. modify /app/onlyoffice/src/build/start/start.backend.docker.sh and change the local_ip like to point to my ip:
local_ip=192.168.0.36
13. /app/onlyoffice/src/build/build.sh
14. /app/onlyoffice/src/build/build.backend.docker.sh
15. cd /app/onlyoffice/src && yarn build
16. cd packages/client && yarn start-prod &
17. cd ../login && yarn start-prod &
18. cd ../editor && yarn start-prod &
19. http://192.168.0.36:8092

After this everything works, but notifications / invitations to user do not due to the fact that onlyoffice-notify docker container is not running. Here is what is running after my commands:

james@ubuntuserver:~$ sudo docker ps
[sudo] password for james:
CONTAINER ID   IMAGE                                                COMMAND                  CREATED        STATUS                    PORTS                                                                   NAMES
033038d004c3   onlyoffice/4testing-docspace-nodejs-runtime:v1.0.0   "python3 docker-entr…"   45 hours ago   Up 36 hours               5050/tcp, 9834/tcp                                                      onlyoffice-ssoauth
2f481d938b50   onlyoffice/4testing-docspace-dotnet-runtime:v1.0.0   "python3 docker-entr…"   45 hours ago   Up 36 hours (unhealthy)   5050/tcp                                                                onlyoffice-files
d1d0a25ed49e   onlyoffice/4testing-docspace-dotnet-runtime:v1.0.0   "python3 docker-entr…"   45 hours ago   Up 36 hours (healthy)     5050/tcp                                                                onlyoffice-people-server
23a891ad71cc   onlyoffice/4testing-docspace-nodejs-runtime:v1.0.0   "python3 docker-entr…"   45 hours ago   Up 36 hours               5050/tcp                                                                onlyoffice-socket
2fb56bc3e458   onlyoffice/4testing-docspace-dotnet-runtime:v1.0.0   "python3 docker-entr…"   45 hours ago   Up 36 hours (healthy)     5050/tcp                                                                onlyoffice-studio-notify
2225e84040d0   onlyoffice/4testing-docspace-dotnet-runtime:v1.0.0   "python3 docker-entr…"   45 hours ago   Up 36 hours (unhealthy)   5050/tcp                                                                onlyoffice-backup-background-tasks
927fec258fba   onlyoffice/4testing-docspace-dotnet-runtime:v1.0.0   "python3 docker-entr…"   45 hours ago   Up 36 hours (healthy)     5050/tcp                                                                onlyoffice-studio
951d1a6d5263   onlyoffice/4testing-docspace-dotnet-runtime:v1.0.0   "python3 docker-entr…"   45 hours ago   Up 36 hours (healthy)     5050/tcp                                                                onlyoffice-backup
fb4f66835e2c   onlyoffice/4testing-docspace-dotnet-runtime:v1.0.0   "python3 docker-entr…"   45 hours ago   Up 36 hours (healthy)     5050/tcp                                                                onlyoffice-api
f7c5241ea18e   onlyoffice/4testing-docspace-dotnet-runtime:v1.0.0   "python3 docker-entr…"   45 hours ago   Up 36 hours (unhealthy)   5050/tcp                                                                onlyoffice-files-services
dd04a62779c5   onlyoffice/4testing-docspace-proxy-runtime:v1.0.0    "/docker-entrypoint.…"   45 hours ago   Up 36 hours (healthy)     80/tcp, 8081/tcp, 8099/tcp, 0.0.0.0:8092->8092/tcp, :::8092->8092/tcp   onlyoffice-proxy
0cb6103e370b   redis:7                                              "docker-entrypoint.s…"   45 hours ago   Up 44 hours               6379/tcp                                                                onlyoffice-redis
44293d0d1049   rabbitmq:3                                           "docker-entrypoint.s…"   45 hours ago   Up 44 hours               80/tcp, 4369/tcp, 5671-5672/tcp, 15691-15692/tcp, 25672/tcp             onlyoffice-rabbitmq
88c18eee5d1b   onlyoffice/documentserver:latest                     "/app/ds/run-documen…"   45 hours ago   Up 44 hours               80/tcp, 443/tcp                                                         onlyoffice-document-server
3b7b46d32cf5   mysql:8.0.32                                         "docker-entrypoint.s…"   45 hours ago   Up 44 hours               33060/tcp, 0.0.0.0:33060->3306/tcp, :::33060->3306/tcp                  onlyoffice-mysql-server
james@ubuntuserver:~$

So if anyone has a tip on how to start this onlyoffice-notify docker container, please let me know.

**Comment by jlaguma** [comment]
Can one of the devs help with this? It's obviously a bug on Onlyoffice side.
The build/install/docker/docspace.overcome.yml has no entry for onlyoffice-notify container.
I've manually added the entry here:

`  onlyoffice-notify:
    <<: *x-profiles-local
    image: ${Baseimage_Dotnet_Run}
    working_dir: ${BUILD_PATH}/services/ASC.Notify/service/
    command: ["ASC.Notify.dll", "ASC.Notify"]
    volumes:
      - ${SRC_PATH}/ASC.Notify/service:${BUILD_PATH}/services/ASC.Notify/service/
      - ${SRC_PATH}/ASC.Files/service/:${BUILD_PATH}/products/ASC.Files/server/
      - ${SRC_PATH}/ASC.People/service/:${BUILD_PATH}/products/ASC.People/server/
      - ${ROOT_DIR}/config/appsettings.${ENV_EXTENSION}.json:/app/onlyoffice/config/appsettings.${ENV_EXTENSION}.json
      - ${ROOT_DIR}/config/apisystem.${ENV_EXTENSION}.json:/app/onlyoffice/config/apisystem.${ENV_EXTENSION}.json
    depends_on:
      onlyoffice-migration-runner:
        condition: service_completed_successfully`

Basically a copy of "onlyoffice-studio-notify", with "Studio.Notify" replaced with "Notify".
After this nodification, onlyoffice-notify container starts up.
<img width="1083" alt="image" src="https://github.com/ONLYOFFICE/DocSpace/assets/20492291/5e0e79fe-dff8-4bd8-b5fc-b6d0cc39936a">

But things are still not working due to the fact that onlyoffice-notify stuck in restart loop.

Can we get any tips on how to resolve this?

**Comment by jlaguma** [comment]
I managed to fix everything my modifying “volumes” at the end of build/install/docker/docspace.profiles.yml to:
`volumes:
  es_data:
  proxy_log:
  app_data:
  files_data:
  people_data:
  crm_data:
  project_data:
  calendar_data:
  mail_data:`

Now everything works!

**Comment by jlaguma** [comment]
fixed
